src/components/Errors/Error404.tsx

```diff
@@ -5,14 +5,15 @@ import * as css from "./Errors.scss";
 
 export default class Error404 extends React.PureComponent {
 
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
why is this needed? I suspect it is because you didn't figure out a better alternative to any. I think there likely is a better way and would prefer that we find it.